doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,108 @@ Options for configuring NEST
++---------------------------------------------+----------------------------------------------------------------+
+| -Dtics_per_ms=[number]                      | Specify elementary unit of time. [default 1000.0]              |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dtics_per_step=[number]                    | Specify resolution. [default 100]                              |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dexternal-modules=[OFF|<list;of;modules>]  | External NEST modules to be linked in, separated by ';'.       |
+|                                             | [default=OFF]                                                  |
++---------------------------------------------+----------------------------------------------------------------+
+
+Generic build configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Dstatic-libraries=[OFF|ON]                 | Build static executable and libraries. [default=OFF]           |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-optimize=[OFF|ON|<list;of;flags>]    | Enable user defined optimizations. Separate multiple flags by  |
+|                                             | ';'. [default OFF, when ON, defaults to '-O3']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-warning=[OFF|ON|<list;of;flags>]     | Enable user defined warnings. Separate  multiple flags by ';'. |
+|                                             | [default ON, when ON, defaults to '-Wall']                     |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-debug=[OFF|ON|<list;of;flags>]       | Enable user defined debug flags. Separate  multiple flags by   |
+|                                             | ';'. [default OFF, when ON, defaults to '-g']                  |
++---------------------------------------------+----------------------------------------------------------------+
+|-Dwith-intel-compiler-flags=[<list;of;flags>]| User defined flags for the Intel compiler. Separate multiple   |
+|                                             | flags by ';'. [defaults to '-fp-model strict']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-libraries=[<list;of;libraries>]      | Link additional libraries. Give full path. Separate multiple   |
+|                                             | libraries by ';'. [default OFF]                                |
```

Comment:
OFF is not shown to the left. So is the default `OFF` or `''`? Applies also below.